pkg/templates/sprig_wrapper.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package templates
+
+import (
+	"github.com/Masterminds/sprig/v3"
+)
+
+var (
+	SprigFuncMap = sprig.FuncMap()
```

Comment:
Could you please make this a private variable?
```suggestion
	sprigFuncMap = sprig.FuncMap()
```